tests/checkpoint/test_checkpoint.py

```diff
@@ -340,44 +315,52 @@ def test_basic_checkpoint_config_validation(
     checkpoint_config = CheckpointConfig(**config)
     checkpoint: Checkpoint = Checkpoint(
         data_context=context,
-        **filter_properties_dict(
+        **deep_filter_properties_iterable(
             properties=checkpoint_config.to_json_dict(),
             delete_fields={"class_name", "module_name"},
             clean_falsy=True,
         ),
     )
-    assert (
-        filter_properties_dict(
-            properties=checkpoint.self_check()["config"],
-            clean_falsy=True,
-        )
-        == expected_checkpoint_config
+    assert deep_filter_properties_iterable(
+        properties=checkpoint.self_check()["config"],
+        delete_fields={"class_name", "module_name"},
+        clean_falsy=True,
+    ) == deep_filter_properties_iterable(
+        properties=expected_checkpoint_config,
+        delete_fields={"class_name", "module_name"},
+        clean_falsy=True,
     )
```

Comment:
Minor nit, can we do the filtering once instead of in each assert? Something like the following, then using filtered_expected_checkpoint_config for the other asserts:

```suggestion
    filtered_expected_checkpoint_config = deep_filter_properties_iterable(
        properties=expected_checkpoint_config,
        delete_fields={"class_name", "module_name"},
        clean_falsy=True,
    )

    assert deep_filter_properties_iterable(
        properties=checkpoint.self_check()["config"],
        delete_fields={"class_name", "module_name"},
        clean_falsy=True,
    ) == filtered_expected_checkpoint_config
```

Reply:
@anthonyburdi This has been fixed module-wide -- and an additional ineffective code deleted.  Thank you!